### ISSUE TITLE
bottlerocket license file copy in rpm2img is problematic

### ISSUE THREAD

**Issue by webern** [bottlerocket-os/twoliter#80]
This is an exact duplicate of https://github.com/bottlerocket-os/bottlerocket/issues/3177 since the solution touches both GitHub repositories.

### Description

The script, `rpm2img` assumes that we are running in the Bottlerocket monorepo.
During the script execution it copies `LICENSE-MIT` directly from the git repo.
This is problematic because an out-of-tree build may not have these files.

Instead, we should package licenses into an RPM file during the package build step and install them from the RPM package during rpm2img.

In other words, we can no longer do this:
https://github.com/bottlerocket-os/bottlerocket/blob/08497fd1503397a0e02858be10917c34965b7ca8/tools/rpm2img#L233

### Definition of Done

When the build no longer relies on Bottlerocket's license files being being present during rpm2img.
```
id: br_licenses
epic: twoliter_exec
```

Related to https://github.com/bottlerocket-os/bottlerocket/issues/2669
